INTERPRETER.s

```diff
@@ -97,7 +97,7 @@ LOW7		OCT	177
 		MASK	LOW7			# WHERE CCS EDOP WILL HONOR IT NEXT.
 
 OPJUMP		TS	CYR			# LOWWD ENTERS HERE IF A RIGHT-HAND OP
-		CCS	CYR			# CODE IS TO BE PROCESSED.  TEST PREFICES.
+		CCS	CYR			# CODE IS TO BE PROCESSED.  TEST PREFIXES.
```

Comment:
`PREFICES` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1108.jpg)